src/components/Button.js

```diff
@@ -84,12 +100,17 @@ const propTypes = {
 
 const defaultProps = {
     text: '',
+    shouldShowRightIcon: false,
     icon: null,
+    iconRight: Expensicons.ArrowRight,
+    iconFill: '#fff',
```

Comment:
@thesahindia Get the value from a const file.
Everything else looks good.

Reply:
@rushatgabhane, done.

Reply:
Thanks!!